src/engine/common_systems/PhysicsCollisionManagementSystem.cs

```diff
@@ -86,6 +86,40 @@ private void Update(ref Physics physics, ref CollisionManagement collisionManage
         try
         {
             ref var ignoreCollisions = ref collisionManagement.IgnoredCollisionsWith;
+            ref var removeIgnoreCollisions = ref collisionManagement.RemoveIgnoredCollisions;
+
+            // Check if any ignores can be removed.
+            if (removeIgnoreCollisions != null)
+            {
+                if (removeIgnoreCollisions.Count > 0)
+                {
+                    // Collision management must keep detecting for clip-out ignore collisions.
+                    // Maybe throw this before StateApplied check?
+                    collisionManagement.StateApplied = false;
```

Comment:
This seems pretty bad to constantly force re-checking. I think this needs re-thinking. When there is a change to either list that should mean that StateApplied is set to false, so that should be enough to detect when something needs to be removed.